src/event/views/EventPage.vue

```diff
@@ -32,13 +32,15 @@
           />
           <div class="buttons">
             <AppButton
+              :class="'btn-secondary'"
               class="btn"
-              text="Save Response"
+              content="Save Response"
               @update="handleSave()"
             />
             <AppButton
+              :class="'btn-secondary'"
```

Comment:
I think rather than using the `class` prop, we should define a new `variant` prop that lets us toggle between primary and secondary variants:

```vue
<AppButton variant="primary">Primary Button</AppButton>
```

And:

```vue
<AppButton variant="secondary">Primary Button</AppButton>
```

I apologize if it seems a bit nitpicky! But I think it's more maintainable to use `variant` instead of `class` to define the button type. Think about it in terms of some other developer wanting to use this component in the future. If all they have is the design, then using the variant prop makes it more explicit which button type the button will display as. If we're using class, the developer has to know what class corresponds to which design, which means they would have to peek at its implementation — a sign of bad API design.

Hope that makes sense! Ping me if none of this does.